demos/python_demos/speech_recognition_demo/README.md

```diff
@@ -39,10 +47,10 @@ Here are the essential options:
 
 ```
 usage: speech_recognition_demo.py [-h] -i FILENAME [-d DEVICE] -m FILENAME
-                                  [-b N] [-L FILENAME] [-a FILENAME]
-                                  [--alpha X] [--beta X] [-l FILENAME]
+                                  [-L FILENAME] -p NAME [-b N] [-c N]
+                                  [-l FILENAME]
```

Comment:
there are not described options:
-c N
-l FILENAME

Reply:
Strange, I successfully pushed the commit to my branch, but I still don't see it here in this PR.  https://github.com/AlexeyKruglov/open_model_zoo/commit/ee00797560d8a558c8bc89d8810078363f92e4a3  Trying to understand why is this happening.

Reply:
Now it appeared, without any action from me.

Reply:
ok, that seems was a ctcdecode-numpy installation issue, solved